js/modules/Louisa.js

```diff
@@ -0,0 +1,109 @@
+export default {
+    state: {
+        profile: {
+            username: "your-github-username",
```

Comment:
Put your GitHub username here.

```suggestion
            username: "LouisaOrobiaSilva",
```